### PR TITLE
Wikidata stats

### PR DESCRIPTION
**Pull request by Tristramg**

### CODE REVIEW

File: src/bin/cosmogony.rs [hunk 1]
```diff
@@ -24,6 +24,11 @@ struct Args {
     /// output file name
     #[structopt(short = "o", long = "output", default_value = "cosmogony.json")]
     output: String,
+    /// export as geojson file
+    #[structopt(long = "export-geojson", default_value = "true")]
```
Comment (antoine-de) [inline src/bin/cosmogony.rs]: why not use `output` (as an `Option`)?

File: src/bin/cosmogony.rs [hunk 2]
```diff
@@ -24,6 +24,11 @@ struct Args {
     /// output file name
     #[structopt(short = "o", long = "output", default_value = "cosmogony.json")]
     output: String,
+    /// export as geojson file
```
Comment (nlehuby) [inline src/bin/cosmogony.rs]: currently, the output is not a geojson :thinking:

File: src/bin/cosmogony.rs (hunk 1 above)
Comment (Tristramg) [reply src/bin/cosmogony.rs]: It would change the current behaviour, would you mind?

**Comment by Tristramg** [comment]
I’m working on a PR to avoid loading the geometry. It might work for the world (to be tested)

**Pull request merged**